core/src/main/proto/xtdb/log/proto/log.proto

```diff
@@ -28,6 +28,8 @@ message TrieMetadata {
     int64 max_system_from = 6  [default = -9223372036854775808];
     int64 row_count = 7 [default = 0];
     bytes iid_bloom = 8;
+    repeated string hll_column_name = 9;
+    repeated bytes hyper_log_log = 10;
 }
```

Comment:
I don't if you prefer a composite object here.

Reply:
Is this message the right place to put the HLLs? they're tied to the table, rather than to the trie itself (just happens to be the L0 trie?)

When we come to read them back in, I suspect we'll want them as part of the table-block file, and written cumulatively (so that we don't have to scan the history)